### PR TITLE
Multiple flow editor fix for issue #2086

### PR DESCRIPTION
**Pull request by enigmaticmahesh**

### CODE REVIEW

File: src/components/floweditor/FlowEditor.tsx [hunk 1]
```diff
@@ -277,13 +277,22 @@ export const FlowEditor = (props: FlowEditorProps) => {
...
+          getOrganizationServices();
+          setCurrentEditDialogBox(false);
+        }}
+        handleMiddle={() => {
           setConfirmedNavigation(true);
           history.push('/flow');
         }}
       >
-        <p className={styles.DialogDescription}>Please try again later or contact the user.</p>
+        {/* <p className={styles.DialogDescription}>Please try again later or contact the user.</p> */}
```
Comment (mdshamoon) [inline src/components/floweditor/FlowEditor.tsx:291]: Remove comment if not needed.

**Review by mdshamoon — CHANGES_REQUESTED** [review]

File: src/components/floweditor/FlowEditor.tsx (hunk 1 above)
Comment (enigmaticmahesh) [reply src/components/floweditor/FlowEditor.tsx]: Removed this with this [Commit](https://github.com/glific/glific-frontend/commit/570664b36f545887015cf12542321f3a325be985)

**Comment by mdshamoon** [comment]
@enigmaticmahesh lets make the UI according to the design file: https://www.sketch.com/s/ba323b85-11d7-4687-8a19-681a32132bab/a/agpQmwP#Inspect

I see some extra space between the message and button.

**Comment by enigmaticmahesh** [comment]
@mdshamoon , Removed the space as per the [Review Comment](https://github.com/glific/glific-frontend/pull/2088#issuecomment-1187247433). It's in this [Commit](https://github.com/glific/glific-frontend/pull/2088/commits/5e54f85a7a0f7814638b2771aa6249c421bcdbb5)

**Comment by mdshamoon** [comment]
@kurund fixed the takeover part. Please have a look now.

**Review by kurund — APPROVED** [review]
Looks good and working as expected.

**Pull request merged**